src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        User findUser = userService.findByEmail(email);
+        if (findUser != null) {
+            if (findUser.getPassword().equals(password)) {
+                return true;
+            }
+        }
         return false;
```

Comment:
Rewrite this code without using true/false literals. You can directly return the result of the condition instead of using if/else or ternary operation to return true/false. For example:
  // Instead of this
  if (condition) {
      return true;
  } else {
      return false;
  }
  // You can do this
  return condition;

This makes the code cleaner and more straightforward.